test/test_autograd.py

```diff
@@ -1502,6 +1502,30 @@ def func(x):
         gradcheck(func, [x])
         gradgradcheck(func, [x])
 
+    def test_complex_view_functions(self):
```

Comment:
This is just testing the backward. not the views.

Reply:
yeah that's correct. This file just contains autograd specific tests so the name was just to reference which functions' backwards are being tested here.